buddyboss/buddyboss-platform
# 780 - Added translation support for the file uploading JS

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #780 .

### How to test the changes in this Pull Request:

1. Go to 'Activity' page.
2. Click on media upload icon.
3. You will see the text 'Drop files here to upload'

### Proof Screenshots or Video


https://www.loom.com/share/0ba17a89069b49158b6998ae66f7936e

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
